extra-xfce/thunar-archive-plugin/spec

```diff
@@ -1,5 +1,5 @@
 VER=0.4.0
 SRCS="https://archive.xfce.org/src/thunar-plugins/thunar-archive-plugin/${VER%.*}/thunar-archive-plugin-$VER.tar.bz2"
 CHKSUMS="sha256::bf82fa86a388124eb3c4854249c30712b2922e61789607268ee14548549b3115"
-REL=2
+REL=3
```

Comment:
Please audit dependencies on `xarchiver` to see if it comes with all dependencies necessary to create archives listed in the context menu.